psi4/src/psi4/libmints/pseudospectral.h

```diff
@@ -66,18 +62,13 @@ class PseudospectralInt : public OneBodyAOInt {
     double C_[3];
     /// Recursion object that does the heavy lifting.
     ObaraSaikaTwoCenterVIRecursion potential_recur_;
```

Comment:
If an OS object is still needed as a member, same comment as before: document enough so somebody else can get rid of this when the time is right.